code/network/multiutil.cpp

```diff
@@ -2945,14 +2928,13 @@ void multi_get_mission_checksum(const char *filename)
 
 char multi_unit_to_char(float unit)
 {
+	Verify((unit > 1.0f) && (unit < -1.0f), "Newly implemented function multi_unit_to_char() was passed a value %f, that it cannot handle.", unit);
```

Comment:
here and below:
the Verify macro only supports a single argument.
You could use the old hack/trick of "anding" the check and the message.
The format is not possibly unfortunately. Asserts and their variants like Verify usually only support string literals.

```Verify((unit > 1.0f) && (unit < -1.0f) && "Newly implemented function multi_unit_to_char() was passed a value, that it cannot handle.");```

Reply:
Ah. I probably should avoid using a hack in a cleanup.  I'll just switch to Assert.  Whoever wants to implement this function can do it using debug builds.